agent/php_user_instrument.c

```diff
@@ -130,6 +130,10 @@ static nr_php_wraprec_hashmap_t* user_function_wrappers;
 static inline void nr_php_wraprec_lookup_set(nruserfn_t* wr,
                                              zend_function* zf) {
   nr_php_wraprec_hashmap_update(user_function_wrappers, zf, wr);
+  // store the wraprec in the op_array extension for the duration of the request for later lookup
+  // for situation when wraprec is added after first execution of the function
+  ZEND_OP_ARRAY_EXTENSION(&zf->op_array, NR_PHP_PROCESS_GLOBALS(op_array_extension_handle)) = wr;
```

Comment:
check for [trampolining](https://github.com/php/php-src/blob/master/Zend/zend_extensions.c#L281-L289) before using zend_op_array_extension.

Does `zf` need a NULL check before dereferencing?